packages/components/src/theme/ComposerTheme/Components/Progress.ts

```diff
@@ -0,0 +1,37 @@
+import { progressAnatomy as parts } from '@chakra-ui/anatomy';
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme';
+import {
+  getColor,
+  PartsStyleFunction,
+  StyleFunctionProps,
+  SystemStyleObject,
+} from '@chakra-ui/theme-tools';
+
+const filledStyle = (props: StyleFunctionProps): SystemStyleObject => {
+  const { theme: t, isIndeterminate } = props;
+
+  const bgColor = 'secondary-ui-default';
```

Comment:
According to the figma file the background color of the progress should be `background-static`